### ISSUE TITLE
test video problem

### ISSUE THREAD

**Issue by Jessespace** [facebook/C3D#383]
I have run through the program and replaced it with my own data set. But I can only output the accuracy of the training set and the test set. if I want to get the test result of single video,what should I do?
